content/doc/book/managing/change-system-timezone.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: section
+title: Change System Time Zone
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+= Change System Time Zone
+
+If you cannot change the time zone of your server, you can force jelly to use a given time zone for formatting time stamps.
+
+You need to start your Jenkins with the following java system property:
+
+[source]
+----
+java -Dorg.apache.commons.jelly.tags.fmt.timeZone=TZ ...
+----
+
+where TZ is a java.util.TimeZone ID ("Europe/Paris" for example).
+
+_Note that `+user.timezone=Europe/Paris+` will work as well, but it can interfere with other contexts._
+
+If running Jenkins via a system package, this can be accomplished by
+setting JAVA_ARGS in your /etc/default/jenkins (Debian):
+
+[source]
+----
+JAVA_ARGS="-Dorg.apache.commons.jelly.tags.fmt.timeZone=America/New_York"
+----
+
+or /etc/sysconfig/jenkins (Red Hat):
+
+[source]
+----
+JENKINS_JAVA_OPTIONS="-Dorg.apache.commons.jelly.tags.fmt.timeZone=America/New_York"
+----
+
+or, if that doesn't work:
+
+[source]
+----
+JENKINS_JAVA_OPTIONS="-Duser.timezone=America/New_York"
+----
+
+On FreeBSD, the file to edit is /etc/rc.conf, and the option to use is:
+
+[source]
+----
+jenkins_java_opts="-Dorg.apache.commons.jelly.tags.fmt.timeZone=America/Indiana/Indianapolis"
+----
+
+On windows, edit `%INSTALL_PATH%/jenkins/jenkins.xml`. Put `-Dargs` before `-jar`:
+
+[source]
+----
+<arguments>-Xrs -Xmx256m -Duser.timezone="Europe/Minsk" -Dhudson.lifecycle=hudson.lifecycle.WindowsServiceLifecycle -jar "%BASE%\jenkins.war" --httpPort=8080</arguments>
+----
+
+You can also set it from the link:https://www.jenkins.io/doc/book/managing/script-console/[Jenkins Script Console] on a live system without the need for a restart.
+This can also be included in a link:todo-replace-by-actual-path[Post-initialization script] to make it permanent.
```

Comment:
I believe you want a link to the groovy scripts page and we probably want an example script that will perform the time zone change.

```suggestion
This can also be included in a link:/doc/book/managing/groovy-hook-scripts/[Post-initialization script] to make it permanent.
```

Reply:
Would an example script be necessary? The script would just be a duplication of the groovy snippet just below.